presto-docs/src/main/sphinx/connector.rst

```diff
@@ -39,6 +39,7 @@ from different data sources.
     connector/scylladb
     connector/singlestore
     connector/sqlserver
+    connector/hana
```

Comment:
Suggest moving this to line 20 to put HANA in alphabetic order in the list. See the first item in [Code Style](https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md#code-style) in CONTRIBUTING.md.